source/llassetgen/include/llassetgen/Atlas.h

```diff
@@ -53,12 +54,13 @@ namespace llassetgen {
         Image atlas{packing.atlasSize.x, packing.atlasSize.y, DistanceTransform::bitDepth};
         atlas.fillRect({0, 0}, atlas.getSize(), DistanceTransform::backgroundVal);
 
-        auto rectIt = packing.rects.begin();
-        for (; imgBegin < imgEnd; rectIt++, imgBegin++) {
-            Image distField{imgBegin->getWidth(), imgBegin->getHeight(), DistanceTransform::bitDepth};
-            distanceTransform(*imgBegin, distField);
+#pragma omp parallel for
+        for (int i = 0; i < std::distance(imgBegin, imgEnd); i++) {
+            Image distField{imgBegin[i].getWidth(), imgBegin[i].getHeight(), DistanceTransform::bitDepth};
```

Comment:
if `imgBegin[i]` does not change and is accessed multiple times, use a reference to that item ... `auto & data = imgBegin[i]`  ...